### PR TITLE
UEFI: Show initialization errors in UpdateError

### PR DESCRIPTION
**Pull request by superm1**

### CODE REVIEW

**Comment by hughsie** [comment]
I've reworked these a little; I hope that's okay. If they still look okay to you, we can squash them down before merging.

File: plugins/uefi/fu-plugin-uefi.c [hunk 1]
```diff
@@ -622,16 +610,52 @@ fu_plugin_coldplug (FuPlugin *plugin, GError **error)
...
+	} else {
+		if (!fu_plugin_uefi_delete_old_capsules (plugin, error))
+			return FALSE;
+		if (!fu_uefi_vars_delete_with_glob (FU_UEFI_VARS_GUID_FWUPDATE, "fwupd-*", error))
+			return FALSE;
+	}
+
+	/* save in report metadata */
+	g_debug ("ESP mountpoint set as %s", data->esp_path);
+	fu_plugin_add_report_metadata (plugin, "ESPMountPoint", data->esp_path);
```
Comment (superm1) [inline plugins/uefi/fu-plugin-uefi.c]: As I can tell this is now duplicated with the proxy device registration so this can probably go.

Comment (hughsie) [reply plugins/uefi/fu-plugin-uefi.c]: Are you sure? I thought plugins didn't handle their own registrations anymore?

Comment (superm1) [reply plugins/uefi/fu-plugin-uefi.c]: Oh right, I forgot about that change.  Then yeah this PR looks fine.

File: src/fu-util.c [hunk 2]
```diff
@@ -446,7 +446,8 @@ fu_util_build_device_tree (FuUtilPrivate *priv, GNode *root, GPtrArray *devs, Fw
 {
 	for (guint i = 0; i < devs->len; i++) {
 		FwupdDevice *dev_tmp = g_ptr_array_index (devs, i);
-		if (!(fwupd_device_has_flag (dev_tmp, FWUPD_DEVICE_FLAG_UPDATABLE) || priv->show_all_devices))
+		if (!fwupd_device_has_flag (dev_tmp, FWUPD_DEVICE_FLAG_UPDATABLE) &&
```
Comment (superm1) [inline src/fu-util.c]: Lost the `priv->show_all_devices`

Comment (superm1) [reply src/fu-util.c]: Also it seems that `fwupd_device_has_flag (dev, FWUPD_DEVICE_FLAG_UPDATABLE` should be in `fu_util_is_interesting_device`

Comment (hughsie) [reply src/fu-util.c]: Gahh, copy paste issue. Will fix.

**Pull request merged**